projects/noloan2/app/src/main/java/hasadna/noloan2/LawsuitFormFragment.java

```diff
@@ -0,0 +1,124 @@
+package hasadna.noloan2;
+
+import android.app.DatePickerDialog;
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.CheckBox;
+import android.widget.EditText;
+import android.widget.TextView;
+import java.util.Calendar;
+
+import hasadna.noloan2.protobuf.LawsuitProto.Lawsuit;
+import noloan.R;
+
+public class LawsuitFormFragment extends Fragment {
+  private static final String TAG = "LawsuitFormFragment";
+
+  EditText receivedDate;
+  DatePickerDialog datePickerDialog;
+  Calendar calendar;
+  Button buttonNext;
+  TextView selectCourt;
+
+  @Override
+  public void onCreate(Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+  }
+
+  @Override
+  public View onCreateView(
+      LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+    ViewGroup rootView =
+        (ViewGroup) inflater.inflate(R.layout.lawsuit_fragment_form, container, false);
+
+    receivedDate = rootView.findViewById(R.id.EditText_receivedSpamDate);
+    selectCourt = rootView.findViewById(R.id.textView_selectCourt_FormFragment);
+    buttonNext = rootView.findViewById(R.id.button_formFragment_next);
```

Comment:
The name "buttonNext" is not regular. You didn't name any of the other buttons with a "button__"

Reply:
Changed other Buttons accordingly.

Reply:
Isn't it a bit redundant to have "button__" for every button? we know it is a button by its variable type

Reply:
I agree, though having some of the subject's names relevant to different kind of components thru out the code, I find it easier and faster to understand the type also by its name in this case.

That was the thought.